audit-github-repo-settings.py

```diff
@@ -174,7 +174,7 @@ def tests(github_branch, repo, token):
             logging.info(green + msg + '"Require status checks before merging" enabled')
 
             # "Status checks that are required"
-            ci_name = "ci/circleci: build"
+            ci_name = "ci.yml"
```

Comment:
Are we checking if repos have a specific CI file? Why?

Reply:
I updated the required status check from ci/circleci: build to ci.yml because we've switched from CircleCI to GitHub Actions in most of our repositories and most of our GitHub Actions workflows are named ci.yml if not we can change this accordingly.
![image](https://github.com/user-attachments/assets/d91c0b41-a1a1-45bc-8b28-23067e748cdb)